renderPage.js

```diff
@@ -108,6 +93,28 @@ function renderPage(Page) {
       }
       return i === -1
     }
+
+    updateUserInfo() {
+      doApiRequest('/clubs/?format=json')
+        .then(resp => resp.json())
+        .then(data => this.setState({ clubs: data }))
+      doApiRequest('/settings/?format=json').then((resp) => {
+        if (resp.ok) {
+          resp.json().then(data => {
+            this.setState({
+              authenticated: true,
+              favorites: data.favorite_set.map(a => a.club),
+              userInfo: data,
+            })
+          })
+        } else {
+          this.setState({
+            authenticated: false,
+            favorites: JSON.parse(localStorage.getItem('favorites')) || [],
```

Comment:
Would be good to abstract all operations on local store to their own file and we just call a function like `store.getFavorites()`

If you'd rather not do that in this PR, can you make a Trello card so we make sure we do it eventually?

Reply:
to trello